### PR TITLE
Use ENV for configure exclusively

### PR DESCRIPTION
**Pull request by timurvafin**
* Remove `app_config` shortcut
* Add `HOST` variable
* Add `MAILER_SENDER_ADDRESS` variable

We should configure application using only ENV variables.
No need to add additional layer of complexity with `app_config`.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,33 +1,39 @@
...
+# Database name prefix
 # DATABASE_NAME=paste_database_prefix_here
 
-# set Rollbar key for the app:
+# Set Rollbar key for the app
 # ROLLBAR_KEY=your_key_here
 
-# enable basic auth to close the app from unauthorized viewers:
+# We send devise email using this "from" address
+MAILER_SENDER=noreply@example.com
```
Comment (dmitrytrager) [inline .env.example]: maybe 'address' is better than 'sender' here

**Comment by dmitrytrager** [comment]
:ok: except tests

**Comment by ruslankhaertdinov** [comment]
Looks :+1:

**Comment by arkadiybutermanov** [comment]
:ok:

**Pull request merged**